machine/alphanetworks/alphanetworks_snx60a0_486t/i2ctools/001-i2ctools-sys-eeprom-alpha-snx60a0-486t.patch

```diff
@@ -0,0 +1,17 @@
+Patch to add sys_eeprom information
```

Comment:
Using i2ctools for the onie-syseeprom command is no longer supported since March 16th 2015.  See this message to the mailing list:

http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html

We only use the busybox implementation of onie-syseeprom going forward.  The message to the group describes how to use the busybox method and a pointer to an example.

Let me know if you have any problems with that.